src/App.vue

```diff
@@ -89,6 +90,7 @@
     </v-snackbar>
 
     <v-dialog
+      :id="'dialog-'+dialog.title.replace(' ','')"
```

Comment:
Let's strive for consistency in code stile. You can look at the code that's already written to get an idea on guidelines.

- Space after a comma.
- For property binding, use `"` as the outer string delimiter and'`' inside. 
- If doing complex string interpolation and concatenations in the template, you can use template literals: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
sounds good @Maurier sorry to ask you to look at 💩. Will use template literals when things get complicated